src/Workspaces/Core/Portable/WorkspacesResources.resx

```diff
@@ -456,4 +456,7 @@
   <data name="Exceptions" xml:space="preserve">
     <value>Exceptions:</value>
   </data>
+  <data name="ImmutableArrayIncorrectlyInitialized" xml:space="preserve">
+    <value>ImmutableArray was incorrectly initialized, consider using ImmutableArray&lt;T&gt;.Empty</value>
```

Comment:
@jasonmalinowski It is shown in Visual Studio as an adornment indicating that a codefix has been disabled for throwing an exception.  The activity log will contain the stack trace and exception that was thrown.

@CyrusNajmabadi Good point, I'll update the message.